src/communication_interface.cc

```diff
@@ -437,3 +454,28 @@ void CommunicationInterface::HandlePause(
     pause_data_.paused_ = true;
   }
 }
+
+// TODO(@syler): this could be a different message type, pause_cmd is probably
+// not the best here
+void CommunicationInterface::HandleSimDriverEventTrigger(
+    const ::lcm::ReceiveBuffer*, const std::string&,
+    const robot_msgs::pause_cmd* cmd_msg) {
+  // bool desired_state {cmd_msg->data};
+  auto desired_event {cmd_msg->source};
```

Comment:
i thought source in the message meant `hostname+cmd`?

Reply:
That was for pause commands send from `cmd`, this is entirely unrelated. They just happen to use the same message type